gpii/node_modules/lifecycleManager/test/LifecycleManagerUtilsTests.js

```diff
@@ -0,0 +1,317 @@
+/*!
+GPII Lifecycle Manager Tests
+
+Copyright 2019 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
```

Comment:
I think this can be a browser test. Keeping the LM browser-clean has somewhat become relevant again with the concept that we may try to embed it in a browser extension, e.g. for UIO+, or as a ChromeOS extension